modules/openthread/platform/crypto_psa.c

```diff
@@ -287,6 +295,160 @@ otError otPlatCryptoHmacSha256Finish(otCryptoContext *aContext, uint8_t *aBuf, s
 	return psaToOtError(psa_mac_sign_finish(operation, aBuf, aBufLength, &mac_length));
 }
 
+
+otError otPlatCryptoHkdfInit(otCryptoContext *aContext)
+{
+	otError error = OT_ERROR_NONE;
+	psa_key_derivation_operation_t *operation;
+
+	if (!checkContext(aContext, sizeof(psa_key_derivation_operation_t))) {
+		error = OT_ERROR_INVALID_ARGS;
+		goto exit;
+
+	}
+
+	operation = (psa_key_derivation_operation_t *) aContext->mContext;
+
+	*operation = psa_key_derivation_operation_init();
```

Comment:
It's better to `memset` operation to 0. If `operation` is large on a given platform, initializing it with `psa_key_derivation_operation_init()` may needlessly increase the stack usage (we were optimizing it in other functions in the past).